README.md

```diff
@@ -36,7 +36,7 @@ SKYNET_ACCOUNTS_LOG_LEVEL=trace
 
 Meaning of environment variables:
 
-* ACCOUNTS_EMAIL_URI is the full email URI (including credentials) for sending emails.
+* ACCOUNTS_EMAIL_URI is the full email URI (including credentials) for sending emails. example `ACCOUNTS_EMAIL_URI=smtps://hello@gmail.com:MYSUP3R$TRONGPW@smtp.gmail.com:465/?skip_ssl_verify=false`
```

Comment:
```suggestion
* ACCOUNTS_EMAIL_URI is the full email URI (including credentials) for sending emails. example `ACCOUNTS_EMAIL_URI="smtps://hello@gmail.com:MYSUP3R$TRONGPW@smtp.gmail.com:465/?skip_ssl_verify=false"`
```

Reply:
We stopped using quotes around variable values in `.env`. They were causing some issues.